### DISCUSSION TITLE
Redshift-distance function, "linear in z" vs "ln(1+z)"

### DISCUSSION THREAD

**Discussion by RedshiftDrift** [a-cosmology-group/acg#227] [Redshift Models]
*I am transferring the debate between @sahil5d and Eric Lerner to this Discussion Forum.*

***-------- Forwarded Message --------
Sat, 1 Jun 2024 17:04:20 +0000 Sahil Gupta via Cosmology Group***

Can we return to the study of one photon's journey?

> Sahil: According to the d = z * c/H function, light traveling 200 Mly doesn't experience the same redshift as light traveling 100 Mly, then 100 Mly. That is a problem.
> Louis: That would be a problem only for a tired-light mechanism.  The "linear-redshift" relationship (let's call it "L-redshift") is not the result of a tired-light mechanism..... A few proposed redshift mechanisms don't follow the ln(1+z) function... not by a property of light, but by a property of space (expansion, aether, gravitational potential, absolute position, etc.) or matter (mass evolution, etc.) or something else that is linearly dependent on distance or time... "new physics"... or "evolution of something".

Actually, the problem remains for those other attempted explanations for
d = z * c/H
≡  f_obs / f_emit = 1/(1 + d/(c/H)).

If an attempted explanation has f(2d) ≠ f(d) * f(d), then it can't explain the properties of the electromagnetic pulse traveling intergalactic space step by step, whether megalightyear by megalightyear or as granularly as centimeter by centimeter.

> Eric: Actually, our statistically detailed comparison shows an exact linear relationship with no evolution.  Shown in our workshop video and we are working on the paper.

Can we [...] tackle this question first:

200 Mly away from Earth, a 500 THz photon is emitted. What is its frequency when it reaches the Earth? Also, what was its frequency at the midpoint, after having traveled 100 Mly?

Sahil Gupta

**Comment by RedshiftDrift** [comment]
***-------- Forwarded Message --------
Sun, 2 Jun 2024 04:16:52 Eric Lerner via Cosmology Group***

> Sahil: According to the d = z * c/H function, light traveling 200 Mly doesn't experience the same redshift as light traveling 100 Mly, then 100 Mly. That is a problem.

No, wrong. A photon emitted let us say 4Gpc away, at a redshift of 1, is measured by the observer on earth to have a frequency of 1/2 the original frequency. A photon emitted 8Gpc away at a red shift of 2, is measured on earth to have a frequency 1/3 the original frequency. A photon emitted at 8Gpc and measured at 4 Gpc is measured to have 1/2 the original frequency.

There is no problem here. You would need to absorb the original photon and re-emit a new photon if you want to measure the frequency from a two-step travel--first 4 Gpc the another 4 Gpc. Since the TL mechanism assumes that some process happens to the photons as they travel through space, there is an expectation that the process starts with the emission of the photon. So there is no expectation that the photon emitted from 4 Gpc will have the same frequency as the one emitted at 8 Gpc when they arrive at earth. They have different ages.

Even without a linear TL mechanism, these two processes are different. The first photon can interfere with another photon emitted from the source at 8 Gpc. The second photon can't.

Energy traveling by emission and absorption step-by-step across the universe is NOT what we observe. We observe radiation that has traveled unimpeded the full distance from the source. That is why we can--and scientists have--formed interference patterns from distant galaxies.

**Reply by RedshiftDrift** [reply]
***-------- Forwarded Message --------
Sun, 2 Jun 2024 12:53:19 Sahil Gupta via Cosmology Group***

> A photon emitted let us say 4Gpc away, at a redshift of 1, is measured by the observer on earth to have a frequency of 1/2 the original frequency.

According to d=z*c/H and if H = 1/(13.05 Gy), sure.

> A photon emitted 8Gpc away at a red shift of 2, is measured on earth to have a frequency 1/3 the original frequency.

According to d=z*c/H.

> A photon emitted at 8Gpc and measured at 4 Gpc is measured to have 1/2 the original frequency.

Ok.

> There is no problem here.

There is a problem. According to d=z*c/H, the photon traveling 8Gpc dropped to 1/3 of its energy.
But a photon traveling 4Gpc dropped to 1/2 its energy, then traveling another 4Gpc dropped again to 1/2 of its energy, for a total drop to 1/2 * 1/2 = 1/4 of its original energy.

> You would need to absorb the original photon and re-emit a new photon if you want to measure the frequency from a two-step travel--first 4 Gpc the another 4 Gpc... So there is no expectation that the photon emitted from 4 Gpc will have the same frequency as the one emitted at 8 Gpc when they arrive at earth. They have different ages.

I'm not even bringing up measurement, whether with a photosensor or a human eye or a random hydrogen atom in the IGM. I'm just asking about the state of the photon along its journey. Things have a state even if unobserved. Pondering the state of a photon does not change the gigaparsec-traveling photon's energy from 1/3 of its original to 1/4.

Even if a measurement were taken, it's not difficult to fathom, in deep space, an iPhone camera sensor absorbing a blue photon and printing the photon frequency on screen and re-emitting the same color blue photon. Maybe that would take 10 milliseconds? But that's not important. The frequency of the light incoming and outgoing would be the same, and would not cause the incongruous [1/3 for 8Gpc] and [1/4 for 4Gpc 4Gpc] energy drop of a d=z*c/H model.

> Energy traveling by emission and absorption step-by-step across the universe is NOT what we observe. We observe radiation that has traveled unimpeded the full distance from the source.

This is interesting in its own right and doesn't affect criticism of the d=z*c/H model.

Also, Louis, how does the STz mechanism deal with Eric's point about "radiation that has traveled unimpeded the full distance from the source"?

Sahil Gupta

**Reply by RedshiftDrift** [reply]
***-------- Forwarded Message --------
Sun, 2 Jun 2024 16:05:09 Eric Lerner via Cosmology Group***

Key question is method, Sahil. Science is about observing the universe and hypothesizing processes that describe how things change. This allows us to make predictions about how they WILL change.

In our existing understanding of EM, a photon propagating freely in space is a DIFFERENT process than a photon being absorbed and re-emitted. In this existing understanding, the coherence of the the photon is destroyed by absorption and re-emission. The photon of the same frequency that is re-emitted is NOT the same as the photon that was traveling freely. (Frequency is NOT the only property that a photon possess, by any means.)

In the linear TL hypothesis, which could equally be called an "ageing light" hypothesis, the freely traveling and re-emitted photons also differ in their age and that , by this hypothesis, affects their energy loss. So the re-emitted photon arrives with 1/4 of the initial frequency and the freely-traveling photon arrives with 1/3 the initial energy. Different photons, different energies, not the same photon with two energies.

Clear?

**Reply by RedshiftDrift** [reply]
***-------- Forwarded Message --------
Sun, 2 Jun 2024 16:05:09 Louis Marmet via Cosmology Group***

Sahil, Eric,
  [...]

  Sahil forgets that in our existing understanding of QED, the photon has no measurable properties between emission and absorption.  So Sahil's argument is incomplete.

  In our existing understanding of EM, a photon polarizes the medium through which it propagates, causing dispersion.  The observed Dispersion Measure shows that light interacts with the IGM between any galaxy and our telescopes.  There is no such thing as a "freely-traveling photon" because space is filled with plasma.  Eric's argument is also incomplete.

Regards,
Louis

**Reply by RedshiftDrift** [reply]
***-------- Forwarded Message --------
Sunday June 02 2024 09:09:54 PM UTC Sahil Gupta via Cosmology Group***

> Key question is method, Sahil. Science is about observing the universe and hypothesizing processes that describe how things change. This allows us to make predictions about how they WILL change.

I know. And I'm alerting you that [ d = z * c/H ]'s predictions about how things change is baffling in that a photon traveling 8Gpc would lose 67% of its energy, but a photon traveling 4Gpc then 4Gpc would lose 75% of its energy. That is neither observed, nor a consistent scientific prediction.

I'll take it further. Divide 8Gpc into 4 segments, 2Gpc each. Nothing has changed about the total path length or the contents of the column of the path. Your model would predict a photon's energy drops to (2/3)^4 = 16/81 of its original energy, meaning it would lose ~ 80% of its energy.

> the coherence of the the photon is destroyed by absorption and re-emission. The photon of the same frequency that is re-emitted is NOT the same as the photon that was traveling freely. (Frequency is NOT the only property that a photon possess, by any means.)

I know. Still, we're focused on z and frequency. Do you think that a blue photon emitted from a star and a same color blue photon emitted from a smartphone would experience different redshift in gigaparsec travel? Why?

Anyway, I'm not even making a refutation that depends on absorption or re-emission. I am going along with your scenario that the photon's travel is unimpeded.

> In the linear TL hypothesis, which could equally be called an "ageing light" hypothesis, the freely traveling and re-emitted photons also differ in their age and that , by this hypothesis, affects their energy loss. So the re-emitted photon arrives with 1/4 of the initial frequency and the freely-traveling photon arrives with 1/3 the initial energy.

To recap, I am going along with your scenario that the photon's travel is unimpeded. I'm only asking about the state of the photon. The state of the photon in intergalactic space is something a model can predict, like how the state of nuclei in radioactive decay is something a model can predict, and my criticism of [ d = z * c/H ] is that it is brittle to divisions of distance segments.

Sahil Gupta

**Reply by Francois-Zinserling** [reply]
Thank you for sharing.
I'm with @sahil5d on this one.

**Reply by sahil5d** [reply]
> in our existing understanding of QED, the photon has no measurable properties between emission and absorption

Absorption is the measurement, right? So that's like QED saying "the photon has no measurable properties between emission and measurement". That's not much of an insight from QED.

As to a photon having properties between emission and absorption, sure it does. If QED claims the opposite, then I challenge QED. Photons don't vanish during their travel.

**Reply by RedshiftDrift** [reply]
> Absorption is the measurement

No, this is incorrect.  There are other ways to make measurements without absorption.  The polarization of the medium is also a way to make a measurement.

> If QED claims the opposite, then I challenge QED. Photons don't vanish during their travel.

Having no determined property between emission and detection does not mean that the photon vanishes.  Who are you to challenge QED, do you even understand the theory?  Read [Einstein's EPR paper](https://cds.cern.ch/record/405662/files/PhysRev.47.777.pdf) and the [discussions that followed](https://plato.stanford.edu/entries/qt-epr/) over the decades after publication of the paper, read Richard Feynman's [Lectures on Physics](https://www.feynmanlectures.caltech.edu/) and then come back with arguments that are not limited to 19th century physics.

I am not saying that Eric's argumentation is valid, his linear redshift-distance relationship is actually incompatible with 20th century electrodynamics and our knowledge of the composition of the IGM.  However @sahil5d your arguments show a complete lack of understanding of the theory of electrodynamics; you are not going to convince anyone by challenging QED.

**Reply by sahil5d** [reply]
> No, this is incorrect. There are other ways to make measurements without absorption.

What is incorrect? I said absorption is the measurement. You said absorption is a measurement. Fine there are other ways to make measurements (collision, scattering).

> There are other ways to make measurements without absorption. The polarization of the medium is also a way to make a measurement.

What is an example of polarization of the medium with 0 absorption? I suppose a polarizing filter where 100% of the photons go through?

In fact, if polarization of the medium with 0 absorption is a measurement, then that contradicts QED's claim that "the photon has no measurable properties between emission and absorption"!

> Having no determined property between emission and detection does not mean that the photon vanishes. 

You changed it to 'determined property'. Do you mean 'determined property' is the same thing as 'measurable property', or something else, or could you say more plainly 'property'? 

> Who are you to challenge QED, do you even understand the theory?

Well I'm someone who is challenging LCDM when almost all of academia is convinced it's true. So that is one mark of credibility. I'm also someone who is challenging Relativity Theory's Sun going around the Earth, and Maxwell's Retarded Potential Electromagnetism, and now QED claiming that a photon doesn't have properties between emission and absorption.

By the way, QED has many critics including Feynman who thought that QED renormalization was a "shell game", "hocus pocus", and a "dippy process".

> read Richard Feynman's [Lectures on Physics](https://www.feynmanlectures.caltech.edu/)

I read them long ago. And time to time as the occasion arises.

**Reply by sahil5d** [reply]
@RedshiftDrift have you seen Unzicker's criticisms of QED, https://www.youtube.com/watch?v=wvz4MRpq6xs&ab_channel=Unzicker%27sRealPhysics ?

In addition to Feynman, Freeman Dyson criticized QED for its "mathematical inconsistencies" and "infinites swept under the rug".

Also, QED is based on Special Relativity (SR). I've already written why I think SR is false (the Sun doesn't go around the Earth and atmospheric nitrogen molecules don't have the kinetic energy of v=.98c), so with SR debunked, I think QED is debunked too. QED may have a few successful ideas, so they should be factored out and preserved. Those ideas might not even be exclusive to QED (like how the Fresnel coefficient, Fizeau experiment, and Airy experiment findings are not exclusive to SR).

QED, LCDM, BB, SR are one big package.

**Reply by RedshiftDrift** [reply]
@sahil5d those mathematical inconsistencies do not change the results of my calculations, I am far from the regime where these problems occur, and well into the experimentally verified predictions of QED.

A comparison: Newtonian physics is known to have some inconsistencies at large speeds/long times/in high gravity.  Yet it's accurate enough to calculate the kinetic energy of your car driving at 120 km/h on the highway as confirmed by decades of observations.

**Reply by sahil5d** [reply]
I don't understand what you mean by 'my calculations'.? I'm talking about QED at large. QED.. the theory which its inventors said was in the wrong, and the theory involved in scandal after scandal for the electron's magnetic moment.

Also, this thread started with your point about QED saying "the photon has no measurable properties between emission and absorption". I challenged that 3 messages above. Recapping: If polarization of the medium with 0 absorption is a measurement, then that contradicts QED's claim that "the photon has no measurable properties between emission and absorption"!

**Reply by RedshiftDrift** [reply]
The photon has no measurable properties.  But a quantum superposition of photons, which is the correct description of light, does have the measurable properties observed experimentally.

**Reply by HanDeBruijn** [reply]
@RedshiftDrift . Can a single electron be observed? For example if it is released by a single photon with the [Photoelectric effect](https://en.wikipedia.org/wiki/Photoelectric_effect)?

**Reply by RedshiftDrift** [reply]
Yes, the electron is observable but it is not released by a "single photon", as the simplified explanation in the article may lead you to think.  It is a superposition of photons with a narrow energy distribution that produce the superposition of "electron plane waves", also with a narrow energy distribution.  This is what is observed (the theory for that is called the 'second quantization' because both light and electrons are quantized in an interaction).

**Reply by budrap00** [reply]
_Math is not physics._

**Reply by RedshiftDrift** [reply]
No, but physics uses the language of mathematics.

**Reply by budrap00** [reply]
Math is a an indispensable modeling tool for physics. But all models whether qualitative or quantitative are incomplete representations of whatever they are models of. Modelers tend to focus on certain aspects of a phenomenon while ignoring others that seem irrelevant to the aspects under consideration. This can be a useful approach but usually over a limited range. 

It has become a standard procedure in theoretical physics to treat limitations of a model as a consequence of unobservable features that are ascribed to physical reality without benefit of any direct empirical evidence. This spares the model makers of the hard work of making models based on the observed physical phenomena which would require those model makers to come to a physical understanding of the physical system(s) they are attempting to model.

The Sun is not a point mass but it can be treated as such for certain calculations. What is a useful approximation on the scale of the Solar System however is inappropriate for disk galaxies. The fix for that problem was of course Dark Matter.

Dark matter and superposition of states are two examples among many of this behavioral tic among theoretical physicists where the failures of a model are foisted off on physical reality. The theoreticians then run around proclaiming the great success of their "universal laws" that describe a "universe" that doesn't exist. Coupled with a predilection for _reductio ad absurdum_ reasoning theoretical physics has become a hotbed of unphysical nonsense.

QED turns [waves into particles](https://thisislanduniverse.com/2024/06/25/photons-are-not-particles/) while QFT turns particles into waves rendering quantum theory fundamentally incoherent. Superposition of states is a byproduct "explanation" that attempts to rationalize that incoherence by invoking an unobservable and unphysical condition. _Mathematical models do not and cannot determine the nature of physical reality._

**Comment by tiredlyndon** [comment]
Eric wrote:

_the coherence of the photon is destroyed by absorption and re-emission. The photon of the same frequency that is re-emitted is NOT the same as the photon that was traveling freely.
Energy traveling by emission and absorption step-by-step across the universe is NOT what we observe. We observe radiation that has traveled unimpeded the full distance from the source. That is why we can--and scientists have--formed interference patterns from distant galaxies._

It seems to be agreed that when photons travel through glass they are absorbed, a delay occurs, and a ‘new’ photon emitted. So, if Eric is correct, glass should destroy coherence?
So, lets try an experiment. Shine a laser through a diffraction grating and get an interference pattern.
Put a piece of glass between the laser and the grating and will the interference pattern disappear since the coherence has been destroyed by the glass?
Our photon travels, on average, in glass at two thirds of the speed it does in a vacuum due to the delays between absorption and re-emission.
Better still, diffraction gratings are aluminium evaporated onto the top of a glass substrate. Laser fires onto the grating. If the lines are on the opposite side we will get an interference pattern. Turn it the other way around so that the photons travel through glass first and it should (according to Eric, disappear.

**Comment by tiredlyndon** [comment]
Eric wrote:
the coherence of the photon is destroyed by absorption and re-emission. The photon of the same frequency that is re-emitted is NOT the same as the photon that was traveling freely.
Energy traveling by emission and absorption step-by-step across the universe is NOT what we observe. We observe radiation that has traveled unimpeded the full distance from the source. That is why we can--and scientists have--formed interference patterns from distant galaxies.
It seems to be agreed that when photons travel through glass they are absorbed, a delay occurs, and a ‘new’ photon emitted. So, if Eric is correct, glass should destroy coherence?
So, lets try an experiment. Shine a laser through a diffraction grating and get an interference pattern.
Put a piece of glass between the laser and the grating and will the interference pattern disappear since the coherence has been destroyed by the glass?
Our photon travels, on average, at two thirds of the speed it does in a vacuum due to the delays between absorption and re-emission.
Better still, diffraction gratings are aluminium evaporated onto the top of a glass substrate. Laser fires onto the grating. If the lines are on the opposite side we will get an interference pattern. Turn it the other way around so that the photons travel through glass first and it should (according to Eric, disappear.

**Reply by Francois-Zinserling** [reply]
@tiredlyndon 
Light slowing through clear glass is an electromagnetic effect, and not due to absorption and re-emission. In general the photons in visible light do not have enough energy to excite an electron (in glass) to a higher level.  

On a classic level all the photons (of a monochromatic source) are slowed in glass equally but on a quantum level there are probabilities to deal with for each photon and my thinking is the interference pattern will be distorted or even destroyed because photons are not exiting phase-coherent, and phase is critical for interference. 
So in my opinion glass will destroy the interference but I don't offer this in support of Lerner.

This does not align with Lerner's thinking, because in glass the wave slows down but the wavelength also shortens. So the photon energy remains $E = hc/λ$ even as $c$ changes. He is looking for a phenomenon that causes frequency to decrease (longer wavelength), while the speed of light remains unchanged, and the photon loses energy. In my opinion this can only happen if the photon did work along the way, and I cannot see where this happens.

**Reply by RedshiftDrift** [reply]
> Light slowing through clear glass is an electromagnetic effect, and not due to absorption and re-emission

This is correct @Francois-Zinserling, there is no 'absorption - re-emission' process in the propagation of light through a transparent medium.  @tiredlyndon uses obsolete 19th-century physics.  It is known that the reduced velocity is caused by the polarisation of the medium which causes a phase delay in the propagation of the wave.

>Put a piece of glass between the laser and the grating and will the interference pattern disappear since the coherence has been destroyed by the glass?

 No, glass lenses are commonly inserted inside interferometers without destroying the interference pattern, you have this wrong @tiredlyndon. The absence of 'absorption - re-emission' means there is no scattering and no distortion of the wavefront, the interference pattern is not distorted by glass or any transparent medium (like the plasma of the IGM).  In terms of photons, they interact [*coherently*](https://www.cambridge.org/core/books/optical-coherence-and-quantum-optics/F8CB94C70FA64CD3FB60890CA2048168) by polarising the transparent medium. (Read the first two chapters of [Nonlinear Optics of free Atoms and Molecules](https://www.semanticscholar.org/paper/Nonlinear-Optics-of-Free-Atoms-and-Molecules-Hanna-Yuratich/1c372556ae9e8b8d81247eb4f00261027f6e5945) for an explanation of this.)

  You have to go beyond 19th-century physics to understand that photon 'absorption' and 'emission' are *only two* of many types of processes describing the interaction of light with matter.  I also recommend reading Richard Feynman's [Lectures on Physics](https://www.feynmanlectures.caltech.edu/)

> This does not align with Lerner's thinking, because in glass the wave slows down but the wavelength also shortens.

This is not exactly what Eric argues @Francois-Zinserling.  He argues for a "yet unknown ageing light" effect on ***freely traveling photons*** (not photons propagating in a medium with an index of refraction.)  Eric's mistake is to think that interference patterns only appear for photons traveling unimpeded, which is completely wrong.

Photons do polarize the plasma of the IGM (as demonstrated by dispersion in radio signals), so the photons **cannot** travel unimpeded between the galaxy and our telescope.  A hypothetical "ageing effect on unimpeded photons" can only happen over limited distances, over which photons do have properties according to QED.  Then, @sahil5d's initial argument

> $1 + z(200 \mbox{ Mly})  \equiv [1 + z(100 \mbox{ Mly}) ] * [1 + z(100 \mbox{ Mly}) ]$

makes sense in the context of 20th-century electrodynamics, and falsifies the possibility of a linear redshift-distance relationship.

This discussion won't go anywhere until the arguments put forward are compatible with established 20th-century electrodynamics.
And it won't go anywhere until Eric joins the forum...

**Reply by HanDeBruijn** [reply]
>$1+z(200\text{ Mly})=\left[1+z(100\text{ Mly})\right]*\left[1+z(100\text{ Mly})\right]$

That is quite similar to [an argument](https://github.com/orgs/a-cosmology-group/discussions/188#discussioncomment-7666948) of mine in thread 188.

**Comment by mikehelland** [comment]
If we're talking about luminosity distance, it seems pretty clear to me it has to be $(1+z) \frac{c}{H} \log (1+z)$:

![image](https://github.com/a-cosmology-group/acg/assets/19311107/a7ac345c-60cf-4981-9a71-777e527ae795)

https://mikehelland.github.io/hubbles-law/other/supernovae.htm

However, that doesn't necessarily mean the physical distance is $\frac{c}{H} \log (1+z)$. That's what you get using the standard luminosity distance calculation. In my model, that's the light travel time distance. You get it simply by adding time dilation like this:

$$\frac{c}{H}\int_0^z \frac{1}{1+z'}dz'$$

The actual physical distance is:

$$\frac{c}{H}  \frac{z}{1+z}$$

Interestingly, if you decided to quantify redshift as negative blueshift, like this:

$$1+b = \frac{1}{1+z} = \frac{f_{obs}}{f_{emit}}$$

So when $z=1$ you get $b=-0.5$. Then the physical distance is:

$$-b \frac{c}{H}$$

So it ends up being linear there. Just not with z.

**Reply by HanDeBruijn** [reply]
@mikehelland .
We have been going through this quite [a while ago](https://github.com/orgs/a-cosmology-group/discussions/160#discussioncomment-7863607).
When combining the above comment with (my version of) the [Variable Mass Theory](https://www.youtube.com/watch?v=T7jVviVBzKs), it is found that the maximally possible redshift in _your_ universe is $z_{max}=\exp(H/c.c/H)-1=\exp(1)-1\approx 1.7$ , which is a passed station.

**Reply by mikehelland** [reply]
What you seem to be saying is that combining our two models results in a third model, which none of us advocate so I don't find the argument all that interesting. I'm not saying anything against your theory, BTW. Just that yours and mine are different and that's OK.

**Reply by HanDeBruijn** [reply]
>and that's OK.

It's **not** OK. Your so-called _physical distance_ is at best _a function of_ the **observed** physical distance, which is given by the luminosity distance divided by $(1+z)$. Or, as you have displayed yourself in the header part of thread 188:

$$
d_{ltt} = \frac{c}{H_0} \log (1+z)
$$

This has been discussed at length in the thread, [especially here](https://github.com/orgs/a-cosmology-group/discussions/188#discussioncomment-7661530). The discrepancy with reality is caused by your "Minkowski time" $\tau$ which is not attached to any physical phenomenon. It's purely mathematical and it does not correspond with any timing mechanism (clock). $c/H_0$ is not the maximally observable distance and it does not correspond with $z=\infty$.

**Reply by mikehelland** [reply]
My model can be computed fully by this:

    data = []

    H = 70
    H = H / 3.08e19 * 60 * 60 * 24 * 365 * 1e6
    c = 1
    z = 0
    t = 0
    x = 0

    while z < 20:
        t -= 1
        x += c / (1 + z)
        z = c / (c - H * x) - 1

        data.append([t, x, z])

    print(z)

t and x are as described and it goes to as high a z as you want.

**Comment by ExpEarth** [comment]
I have a different take on this problem.  I suggest that the initial premise of a single photon traveling unhindered from a star to our telescope here is in error.  The CMB temperature curve tells us why.  The CMB temperature at a position A in space follows the relationship T = T0(1+z), where T0 is our temperature here.  Now the energy density of the CMB at that point A is proportional to T^4, while the photon number density is proportional to T^3.  So the energy _per photon_ at A is proportional to T.  This means that the average energy and thus frequency of CMB photons at A is proportional to (1+z).  If you were to suppose naively that the CMB photons originating at A all reached us here on Earth, you could mistakenly think that there is a linear relationship of the single photon's energy with distance.  This would be in keeping with Eric's stance.  But In fact the CMB photons seen here cannot be the same ones as originating at A.  The number density here at our position is much reduced.  Ruling out cosmological expansion, the only other thing that can explain this is that photon energy is redistributed amongst photons during the journey, in such a way that the overall number is reduced.  The easiest way to imagine this if photon is transferred to spacetime, which then redistributes the energy.

The situation with starlight is similar.  Starlight is composed of photons from electronic emissions of countless atoms.  A single photon might possibly travel all the way to us.  But at the stellar surface the density of photons is so great that it has to be organized in waves, each containing countless photons.  At the star these waves are actually blackbody radiation.  As waves of starlight traverse space, their photon number must diminish likewise, with energy being redistributed amongst the photons.  So the idea that a single photon is emitted from a star and travels unhindered all the way to us is simply false.  The starlight is no longer blackbody radiation by the time it gets here, but its photons are not the same as what were emitted.  A more complete description of all this can be found in #204

@Francois-Zinserling 
`He (Eric) is looking for a phenomenon that causes frequency to decrease (longer wavelength), while the speed of light remains unchanged, and the photon loses energy. In my opinion this can only happen if the photon did work along the way, and I cannot see where this happens.`

I think you are right and it can happen if photon energy is being transferred to spacetime.  I argue in my paper that this transfer is what gives rise to the gravitational force.  We can call it a tired light mechanism if we choose.  At this point it becomes more complicated though.

**Reply by HanDeBruijn** [reply]
@ExpEarth .
>So the idea that a single photon is emitted from a star and travels unhindered all the way to us is simply false. [ .. ]
We can call it a tired light mechanism [ .. ]

Nope. The idea that **there is no TL** and that **light** becomes **not tired** might be simply **true**. The [Variable Mass Theory](https://www.youtube.com/watch?v=T7jVviVBzKs) says that photons are **emitted** with higher redshift. And nothing else is happening between emission, and absorption (by us). Simple and straightforward.

>photon energy is being transferred to spacetime. [ .. ] this transfer is what gives rise to the gravitational force.

Sorry Matt, IM(NS)HO there is no way to make sense of that.

**Comment by tiredlyndon** [comment]
Hello Louis,
_This is correct @Francois-Zinserling, there is no 'absorption - re-emission' process in the propagation of light through a transparent medium. @tiredlyndon uses obsolete 19th-century physics. It is known that the reduced velocity is caused by the polarisation of the medium which causes a phase delay in the propagation of the wave._
_Nah, you use phase velocity, NTL uses group velocity (the physically correct interpretation of speed of light).
Phase velocity is, in many cases greater than the speed of light in a vacuum! What would Einstein make of that!
What matters is the speed at which the energy travels - group velocity, and this is what NTL is based on.
Look at Dispersion measure data. One clearly sees a peak as the high energy photons arrive followed by a delay and then another peak as the longer wavelength photons arrive.
(If I could remember how to upload the data printout I would do it.)
The radio telescopes do not record phase but they record the energy arriving.
This is my last post on this matter as it's pretty clear what is going on.
You say phase I say group velocity. How fast the energy travels.
Cheers
Lyndon






_
